main.py

```diff
@@ -68,12 +67,13 @@ def run(driver, username, password, url, shoe_size, login_time=None, release_tim
                 driver.get(url)
             except TimeoutException:
                 LOGGER.info("Page load timed out but continuing anyway")
-
+            
             try:
-                select_shoe_size(driver=driver, shoe_size=shoe_size)
+                select_shoe_size(driver=driver, shoe_size=shoe_size, shoe_type=shoe_type)
             except Exception as e:
-                # If we fail to select shoe size, try to buy anyway
+                # Try refreshing page since you can't click Buy button without selecting size
                 LOGGER.exception("Failed to select shoe size: " + str(e))
+                continue
```

Comment:
I think my original intention was that if you've already set your shoe size preference in your profile (which is preferred), then this step may not even be required. But honestly, I'm not sure if that's even true.

Reply:
I tested the other day with a release, I thought their ajax would update the DOM to display the "Buy $XXX" button (similar to what the SNKRs mobile app does) but that is not the case. Upper timeout could be up to 20 seconds before page refresh which could be critical for a hot new release. I can remove for now, if desired. Also thought about reducing the timeout for each to 5 seconds, I'll test that on next release. 

I'll test clicking the "Buy" button without shoe size selection, wasn't sure if that was a feature for the Nike site (works for SNKRs mobile app). Perhaps adding another parameter to skip shoe size/type selection if this is the case?

Reply:
@alfredog1976 I've noticed the Nike is now just popping up a modal with an Error. Have you experienced this? I was curious if their bot protection is picking something up here.

Reply:
@ericruiz-developer I've noticed that too in my testing when I'm manually signing in. Refreshing the page 1 or more times seems to alleviate it. Haven't seen that error when running the bot though.

Reply:
This does happen with bot. One possible way to fix, create another loop for login logic and add a shorter timeout (10-20 seconds) at the myAccount:greeting checkpoint to force a refresh.

Reply:
Yeah, I actually haven't had the error when browsing manually. Only with the bot. I've been looking into it and it seems sites might be picking up the browser drivers. I was looking at altering the binary but if a refresh fixes it that would be waaaaay easier, I wonder if you'll be stuck refreshing every page though...

Reply:
I guess that is a fix for a separate PR then? We prolly need to discuss it a little more on Discord.